docs/contribute/versioning.adoc

```diff
@@ -1,57 +1,99 @@
+Of course there is a gray area in this rule. In cases where the documentation
+is vague and permits multiple interpretations we prefer to make the
+documentation explicit and consistent with established behavior. In other
+words: changing documentation is preferred to changing behavior.
 
 === Only careful changes of supporting files (XSD schemas, XSLT)
 
-Only small non-invasive changes should be considered. Make sure old use-cases aren't broken.
+Only small non-invasive changes should be considered. Make sure old use-cases
+aren't broken.
 
-As an example the report XSLT may get a small usability improvement - it will show number of notchecked rules along with all the other information. However we may not remove or replace information from it in the maintenance branch - users may be relying on it being there. Radically moving information around in the report is also discouraged. It's very hard to lay formal rules surrounding this so please use your best judgement.
+As an example the report XSLT may get a small usability improvement - it will
+show number of notchecked rules along with all the other information. However
+we may not remove or replace information from it in the maintenance branch
+- users may be relying on it being there. Radically moving information around
+in the report is also discouraged. It's very hard to lay formal rules
+surrounding this so please use your best judgement.
 
 === All applied changes must be propagated
 
-Bugfixes applied to maintenance branches must be merged up. For example a bugfix applied to maint-1.0 shall be merged to maint-1.2 which then shall be merged to master.
+Bugfixes applied to maintenance branches must be merged up. For example,
+a bugfix applied to `maint-1.2` shall be merged to `maint-1.x` (where `x > 2`)
+which then shall be merged to `master`.
 
 This rule makes sure we always carry bugfixes to new feature releases.
 
-In cases where the bugfix does not make sense in newer feature releases, a noop merge shall be done. For example a fix for functionality that was removed in newer feature release.
+In cases where the bugfix does not make sense in newer feature releases, a noop
+merge shall be done. For example, a fix for functionality that was removed in
+a newer feature release.
 
 == Frozen branches
 
-Maintenance branches can be frozen which further restricts the changes that can be applied to them. Only bugfixes are permitted in frozen branches.
+Maintenance branches can be frozen which further restricts the changes that can
+be applied to them. Only bugfixes are permitted in frozen branches.
 
 == Maintenance status
 
 	* 1.2.x: supported
 	* 1.1.x: unsupported since 2015-10-16
-	* 1.0.x: supported, frozen since 2016-01-07
+	* 1.0.x: unsupported
```

Comment:
I hope I can answer instead of @mpreisler. It was canceled on 2017-06-01. See https://www.redhat.com/archives/open-scap-list/2017-June/msg00000.html

Reply:
Thx, I will mention it there.